src/main/resources/journals/journalList.csv

```diff
@@ -3310,7 +3308,7 @@ Climate Dynamics;Climate Dyn.;;
 Climate of the Past;Clim. Past;;
 Climate of the Past Discussion;Clim. Past Discuss.;;
 Climate Risk Management;Clim. Risk Manage.;;
-Climatic Change;Clim. Change;;
+Climatic Change;Climatic Change;;
```

Comment:
I think the changes in this file needs to be reverted, as they don't seem to be related to this PR (probably a merge problem).

Reply:
Yes, that was result of incorrect branch update.